### DISCUSSION TITLE
change the synchronization setting

### DISCUSSION THREAD

**Discussion by lenzeloty** [tutao/tutanota#3364] [Ideas]
Is it possible to turn off the auto sync (fetching emails) in the android app? I would appreciate such a feature to change the synchronization setting, like switching from auto sync to manual sync (e.g. on starting the app) and change time intervals of the auto sync. This is pretty common for other email clients.

**Comment by charlag** [comment]
Hi
We are not loading your emails in background at all. First of all, you might be not logged in yet. Second, there's not offline yet (#590). So there's nothing to sync for now.

**Comment by jowlo** [comment]
@lenzeloty Do you just not want to see the notifications maybe?
